dpctl/tensor/libtensor/source/copy_numpy_ndarray_into_usm_ndarray.cpp

```diff
@@ -23,6 +23,7 @@
 //===----------------------------------------------------------------------===//
 
 #include <algorithm>
+#include <exception>
```

Comment:
This header seems to be currently unnecessary.

Reply:
This is needed, to be able to `throw std::runtime_error`, but I must have removed validation that strides of NumPy array are divisible by itemsize while removed `*_legacy` functions. I am going to reinstate it in a minute. Thank you for catching it!

Reply:
It would, most likely, compile with that line removed but removal goes against include-what-you-use guideline.

Reply:
But isn't `std::runtime_error` defined in `stdexcept` rather than `exception`?

Reply:
Yes, https://en.cppreference.com/w/cpp/error/runtime_error. So I am going to push a change to replace `<except>` with `<stdexcept>`